.github/workflows/ms.healthcareapis.workspaces.yml

```diff
@@ -18,14 +18,14 @@ on:
       - main
     paths:
       - '.github/actions/templates/**'
+      - '.github/workflows/template.module.yml'
       - '.github/workflows/ms.healthcareapis.workspaces.yml'
       - 'modules/Microsoft.HealthcareApis/workspaces/**'
       - 'utilities/pipelines/**'
       - '!utilities/pipelines/deploymentRemoval/**'
       - '!*/**/readme.md'
 
 env:
-  variablesPath: 'settings.yml'
   modulePath: 'modules/Microsoft.HealthcareApis/workspaces'
   workflowPath: '.github/workflows/ms.healthcareapis.workspaces.yml'
   AZURE_CREDENTIALS: ${{ secrets.AZURE_CREDENTIALS }}
```

Comment:
Same here. We should be able to remove those too

Reply:
done